libs/core/__deprecated__/renderer/components/rest/menu/expandable-menu/expandable-menu-subitems.component.tsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import { FunctionComponent } from "Core/core/types/function-component.interface"
+import { MenuElementItem } from "Core/__deprecated__/renderer/constants/menu-elements"
+import {
+  LatestSubitemBranchMarker,
+  LatestSubitemBranchMarkerWrapper,
+  SubitemBranchMarkerHorizontalLine,
+  SubitemBranchMarkerVerticalLine,
+  SubitemBranchMarkerWrapper,
+  SubitemButton,
+  SubitemEmptyBox,
+  SubitemWrapper,
+} from "Core/__deprecated__/renderer/components/rest/menu/expandable-menu/expandable-menu-subitems.styles"
+
+const FirstSubitem = () => {
+  return (
+    <SubitemWrapper>
+      <SubitemBranchMarkerWrapper>
+        <SubitemBranchMarkerVerticalLine />
+        <SubitemBranchMarkerHorizontalLine transparent />
+      </SubitemBranchMarkerWrapper>
+      <SubitemEmptyBox />
+    </SubitemWrapper>
+  )
+}
+
+const LatestSubitem: FunctionComponent<MenuElementItem> = (props) => {
```

Comment:
LatestSubitem means "the newest subitem". You probably meant "LastSubitem' 🙂

Reply:
https://github.com/mudita/mudita-center/pull/2329/commits/eb6952e475cd06e46c5606a966003e8a18b1c787